src/lib.rs

```diff
@@ -205,6 +205,10 @@ pub enum Error {
     #[error(display = "Invalid launch argument at index {}", _0)]
     InvalidLaunchArgument(usize, #[error(source)] widestring::NulError<u16>),
 
+    /// Invalid launch arguments.
```

Comment:
Maybe change the description?

Reply:
To what? :)

Reply:
For example to "Launch arguments are not supported"?

Reply:
Just use the same string as the display implementation. It's even more clear.

Reply:
Done in e592829.